src/data_sets.h

```diff
@@ -928,6 +929,20 @@ typedef struct PACKED
     float					magInc;                 
 } inl2_states_t;
 
+// (DID_ROS_COVARIANCE_POSE_TWIST) INL2 - INS Extended Kalman Filter (EKF) state covariance
+typedef struct PACKED
+{
+    /** GPS time of week (since Sunday morning) in seconds */
+	double                  timeOfWeek;
+
+    /** (rad^2, m^2)  EKF attitude and position error covariance matrix lower diagonal in body (attitude) and ECEF (position) frames */
+	float					covPoseLD[21];
```

Comment:
We may want to add/change here "... matrix lower diagonal as row-major array"
Both lines 938 and 941.